### PR TITLE
Updated Certify

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #:* /bcgov/entity#6863

*Description of changes:*
* Includes optional Statements.
* Includes optional MailTo section for incorporations.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

**Review by andrepestana-aot — APPROVED** [review]
Looks good!

File: src/components/Certify/Certify.vue [hunk 1]
```diff
@@ -180,7 +220,7 @@ export default class Certify extends Vue {
   width: 100%;
 }
 
-.certify-clause {
+.certify-clause, .emails-line {
```
Comment (severinbeauvais) [inline src/components/Certify/Certify.vue:223]: What is this new class used for?

**Review by severinbeauvais — APPROVED** [review]
Possible typo but looks great otherwise.

File: src/components/Certify/Certify.vue (hunk 1 above)
Comment (cameron-freshworks) [reply src/components/Certify/Certify.vue:223]: typo indeed. Was renamed, good catch :)
 
Removed in next commit!

**Pull request merged**